app/views/pet_applications/show.html.erb

```diff
@@ -0,0 +1,9 @@
+<h1>All Applicants Information</h1>
+
+  <p>Name: <%= @pet_application.name %></p>
+  <p>Address: <%= @pet_application.address %></p>
+  <p>City: <%= @pet_application.city %></p>
+  <p>State: <%= @pet_application.state %></p>
+  <p>Zip: <%= @pet_application.zip %></p>
+  <p>Phone Number: <%= @pet_application.phone_number %></p>
+  <p>Description: <%= @pet_application.description %></p>
```

Comment:
So good!  I'm so sad we couldn't get the isolated pet_applications owned by the same person.  Makes me think about our many to many relationships and if we did that part correctly.